src/app/project-overview/component.ts

```diff
@@ -83,10 +90,16 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
     private readonly _serviceAccountService: ServiceAccountService,
     private readonly _settingsService: SettingsService,
     private readonly _appConfigService: AppConfigService,
+    private readonly _params: ParamsService,
     private readonly _cookieService: CookieService
-  ) {}
+  ) {
+    if (this.isEnterpriseEdition) {
+      this._quotaService = GlobalModule.injector.get(QuotaService);
+    }
+  }
 
   ngOnInit(): void {
+    this._setProjectId();
```

Comment:
Is this method recalled if I select different project from top nav bar?

Reply:
Looks like we have same issue on clusters page as well (if cluster is set as default landing page)


https://user-images.githubusercontent.com/13975988/186639214-3c751242-cbdd-4d0b-b90f-beaf25b79ad4.mov

Reply:
No, I've updated logic for catering to that condition.

Reply:
@Talha-Jamil-TJ what about clusters page issue?